staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -300,3 +307,61 @@ func (le *LeaderElector) maybeReportTransition() {
 		go le.config.Callbacks.OnNewLeader(le.reportedLeader)
 	}
 }
+
+type LeaderChecker interface {
+	healthz.HealthzChecker
```

Comment:
Go ethos: "a little copy paste is better than a little extra dependency"; can we just copy the function signature and avoid the import? If possible I think the client shouldn't import the server code.

Reply:
Done